src/main.rs

```diff
@@ -195,13 +195,13 @@ pub mod content {
                     }
 
                     let (screen_from, screen_to) = match (screens.get(from), screens.get(to)) {
-                        (Some(screen_from), Some(screen_to)) => (screen_from, screen_to),
+                        (Some(screen_from), Some(screen_to)) => (screen_from.clone(), screen_to.clone()),
```

Comment:
`cargo build` showed a borrow warning here.

Reply:
Not necessary with the other solution.